### ISSUE TITLE
Itemkeep doesn't work with potions

### ISSUE THREAD

**Issue by Alfalik** [twizmwazin/CardinalPGM#668]
+ Video: https://youtu.be/ghSuJsiOvU8
+ Log: http://pastebin.com/da8ZWQDK
+ Cardinal version  62e93bb: http://gyazo.com/9e5eb426d831f3200805e36740fbae3b
+ XML:
```xml
<blockdrops>
    <rule>
        <filter>
            <block>diamond block</block>
        </filter>
        <drops>
            <item enchantment="dig speed:1">gold axe</item>
            <item>tnt</item>
            <item damage="8226" potions="speed:1600:0:false;jump:1600:1:false;fast digging:1600:2:false">potion</item>
        </drops>
    </rule>
</blockdrops>
<itemkeep>
    <item damage="8226" potions="speed:1600:0:false;jump:1600:1:false;fast digging:1600:2:false">potion</item>
    <item enchantment="dig speed:1">gold axe</item>
    <item>tnt</item>
</itemkeep>
```

**Comment by iEli2tyree011** [comment]
Does it work with other items?

**Comment by TiTo-418** [comment]
@iEli2tyree011 Yes

**Comment by iEli2tyree011** [comment]
I've figured out the problem; it's not detecting the right damage values. I'll fix this later tonight.

**Comment by ryanw-se** [comment]
This also seems to be the case for bugs  #679 and #681

**Comment by iEli2tyree011** [comment]
Fixed by eb12bc82689bb82f87326702933e86c215ce9ac3
